docs/quickstart/quickstart.ipynb

```diff
@@ -73,11 +73,13 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
why is this cell changed? Is this necessary?

Reply:
No, I made it such that the output was not scrollable, instead in full. I will revert it to its original state 😄